changes_revision.go

```diff
@@ -445,8 +475,17 @@ func (s *ChangesService) GetPatch(ctx context.Context, changeID, revisionID stri
 		return nil, nil, err
 	}
 
-	v := new(string)
-	resp, err := s.client.Do(req, v)
+	strVal := ""
+	var v *string = &strVal
```

Comment:
Can you help clarify what's the motivation to change `v := new(string)` into  the following code?

```
strVal := ""
var v *string = &strVal
```